src/env/hedged/mod.rs

```diff
@@ -0,0 +1,671 @@
+// Copyright (c) 2017-present, PingCAP, Inc. Licensed under Apache-2.0.
+
+use crossbeam::channel::unbounded;
+use crossbeam::channel::Receiver;
+use log::info;
+use std::fs;
+use std::io::{Read, Result as IoResult, Seek, SeekFrom, Write};
+use std::path::Path;
+use std::path::PathBuf;
+use std::sync::atomic::AtomicU64;
+use std::sync::atomic::Ordering;
+use std::sync::Arc;
+use std::thread;
+use std::thread::JoinHandle;
+
+use crate::env::log_fd::LogFd;
+use crate::env::DefaultFileSystem;
+use crate::env::{FileSystem, Handle, Permission, WriteExt};
+use futures::executor::block_on;
+use futures::{join, select};
+
+mod recover;
+mod runner;
+mod sender;
+mod task;
+mod util;
+
+use runner::TaskRunner;
+use sender::HedgedSender;
+use task::{
+    empty_callback, paired_future_callback, Callback, FutureHandle, SeqTask, Task, TaskRes,
+};
+use util::replace_path;
+
```

Comment:
Refer to https://github.com/tikv/raft-engine/pull/339, this import format should be reformatted.